rebar.config

```diff
@@ -42,3 +42,18 @@
                , {erldis, set, 3}
                , {erldis, mget, 2}
                ]}.
+
```

Comment:
Remove erldis in:
`
%% basho_bench_driver_erldis calls undefined functions that we ignore.
{xref_ignores, [ {erldis_client, connect, 0}
               , {erldis, set, 3}
               , {erldis, mget, 2}
               ]}.
`
and erldis in
`cover_excl_mods, [ basho_bench_driver_erldis`